src/utilities/string-utilities.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+if ( ! function_exists( 'starts_with' ) ) {
+  /**
+   * Alias
+   */
+  function starts_with( $haystack, $needle ) {
+    return ! strncmp( $haystack, $needle, strlen( $needle ) );
+  }
+}
+
+if ( ! function_exists( 'ends_with' ) ) {
+  /**
+   * Alias
+   */
+  function ends_with( $haystack, $needle ) {
+    $haystack_length = strlen( $haystack );
+    $needle_length   = strlen( $needle );
+    if ( $needle_length > $haystack_length )
+      return false;
+    return substr_compare( $haystack, $needle, -$needle_length ) === 0;
+  }
+}
```

Comment:
I'm *trying* (but still failing) to write the tests for functions as I move them over. Just haven't gotten to that one yet.